lib/ancestry/has_ancestry.rb

```diff
@@ -31,7 +36,7 @@ def has_ancestry options = {}
     self.touch_ancestors = options[:touch] || false
 
     # Validate format of ancestry column value
-    validates_format_of ancestry_column, :with => Ancestry::ANCESTRY_PATTERN, :allow_nil => true
+    validates_format_of ancestry_column, :with => ancestry_pattern, :allow_nil => true
```

Comment:
This is where ancestry_pattern is used that I thought would be expensive if calculated over and over.